content/blog/2021-01-26-january-21-community-gems.md

```diff
@@ -0,0 +1,184 @@
+---
+title: January '21 Community Gems
+date: 2021-01-26
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: parallelize your data transfer, 
+  compressed datasets, and DVC pipelines in CI/CD.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: parallelize your data transfer, 
+  compressed datasets, and DVC pipelines in CI/CD.
+picture: 2020-12-30/cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/december-20-gems/606
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Plots
+  - Pipelines
+  - Docker
+---
+
+## DVC questions
+
+### [Q: My dataset is made of more than _a million_ small files. Can I use an archive format, like `tar.gz` with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/798983422965841920)
+
+There are some downsides to using archive formats with DVC, and often we
+discourage it- but let's review some factors to consider, so you can make the
+best choice for your project.
+
+- DVC versions via snapshots, not diffs. So if your `tar.gz` file changes at
+  all- perhaps because you changed a single file before zipping- DVC will create
+  an entirely new copy of the archive for versioning! This is not very space
+  efficient, but if space isn't an issue it might not be a dealbreaker.
+- Because of the way DVC optimizes data transfer, you'll end up transferring the
```

Comment:
you make it a bit too much about DVC ... it's a general problem when archives are being used - it's a single file, and you have to move it, it's very hard to support diff

Reply:
good point